federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -42,9 +42,9 @@ spec:
           mountPath: /srv/kubernetes/
           readOnly: true
       - name: etcd
-        image: quay.io/coreos/etcd:v2.3.3
+        image: gcr.io/google_containers/etcd:3.0.14-alpha.1
         command:
-          - /etcd
+          - /usr/local/bin/etcd
```

Comment:
I don't think this is sufficient. Federated Service is in beta, which means we need to provide an upgrade path. I think we should be doing what kubernetes is doing. See the manifest template here - https://github.com/kubernetes/kubernetes/blob/84df0679671e719971734be09e954c28f7febaf9/cluster/saltbase/salt/etcd/etcd.manifest#L42